src/openms/include/OpenMS/DATASTRUCTURES/FASTAContainer.h

```diff
@@ -199,19 +200,19 @@ class FASTAContainer<TFI_File>
   }
 
   /// is the FASTA file empty?
-  bool empty() const
+  bool empty()
```

Comment:
why do we loose the const here?

Reply:
because of the function atEnd() that is called there. The funcions peek(), tellg(), seekg() are not const :(

Reply:
without knowing the details... could the stream not be declared "mutable"?